README.md

```diff
@@ -29,9 +29,6 @@ app.engine('jsx', require('express-react-views').createEngine());
 Beginning with v0.2, you can now pass options in when creating your engine.
 
 option | values | default
--------|--------|--------
-`jsx.harmony` | `true`: enable a subset of ES6 features | `false`
-`jsx.stripTypes` | `true`: strip [Flow](http://flowtype.org/) type annotations | `false`
 `jsx.extension` | any file extension with leading `.` | `".jsx"`
```

Comment:
If we end up with just one `jsx.*` options, then we can probably just make this top level (would simplify the defaults code anyway).